backend/common/constants.py

```diff
@@ -21,6 +21,7 @@
     "./frontend/playground-frontend/src/backend_outputs/visualization_output/confusion_matrix.png"
 )
 CLASSICAL_ML_RESULT_CSV_PATH = f"ml_results.csv"
+IMAGE_DETECTION_RESULT_CSV_PATH = f"detection_results.csv"
```

Comment:
where is the code that writes object detection results to this file path or is this only on download? @Dashrekker

Reply:
the code is in the write_to_csv function in detection.py